src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -16,6 +16,9 @@
 #include <string.h>
 #include <sys/stat.h>
 #include "coreruncommon.h"
+#if defined(__LINUX__)
```

Comment:
I was conservative since this is only needed for the __LINUX__ implementation for the time being.